Cheza-Dzabala/idove-frontend
# CH: Setup React Repo

#### What does this PR do?
- Introduces necessary files for base setup of the iDove frontend reposiory

#### Description of Task to be completed?

- Create repository
- Setup CI with TravisCi
- Setup Coverage reporting with CodeCove

#### How should this be manually tested?

- Clone this repo
- Run `npm install`
- Run `npm run dev`
- Check for `welcome to iDove` text

#### Any background context you want to provide?
React application setup using `npx create-react-app` 

#### What are the relevant pivotal tracker stories?
##### #170769441

#### Screenshots (if appropriate)
N / A
#### Questions:
N / A
